recipes/oop-ext/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "oop-ext" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dd5982b0d677b585c76996cae5470c469325f9275ae3317fceabdfef2c3329ec
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
We need to `--no-deps` there. See https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip